app/tasks/usage.php

```diff
@@ -120,21 +91,12 @@ function aggregateDatabase(UtopiaDatabase $database, callable $logError): void
     ->task('usage')
     ->param('type', 'timeseries', new WhiteList(['timeseries', 'database']))
     ->desc('Schedules syncing data from influxdb to Appwrite console db')
-    ->action(function (string $type) use ($register, $logError) {
+    ->action(function (string $type) use ($logError) {
         Console::title('Usage Aggregation V1');
         Console::success(APP_NAME . ' usage aggregation process v1 has started');
 
-        $redis = $register->get('cache');
-        $dbPool = $register->get('dbPool');
-
-        $database = $dbPool->getConsoleDB();
-        $pdo = $dbPool->getPDO($database);
-        $database = DatabasePool::wait(
-            DatabasePool::getDatabase($pdo, $redis, '_console'),
-            'projects',
-        );
-
-        $influxDB = getInfluxDB($register);
+        $database = getConsoleDB();
+        $influxDB = getInfluxDB();
```

Comment:
Looks like these functions don't exist ?

Reply:
Usage is a work in progress. Also need help finishing migration.